elastic_apm.libraries.yml

```diff
@@ -0,0 +1,14 @@
+# Real user monitoring javascript.
+rum:
+  remote: https://unpkg.com/@elastic/apm-rum@4.4.2/dist/bundles/elastic-apm-rum.umd.min.js
+  version: 4.4.2
+  license:
+    name: MIT
+    gpl-compatible: true
+  js:
+    https://unpkg.com/@elastic/apm-rum@4.4.2/dist/bundles/elastic-apm-rum.umd.min.js:
+      type: external,
```

Comment:
**CHANGE REQUEST**

Is that `,` needed here? It looks like a mistake for YAML files.

Reply:
Yeah its not needed actually, I have removed it